themes/default/register.tmpl.php

```diff
@@ -100,17 +100,17 @@ function encrypt_password()
 		<?php if ($this->show_password) { ?>
 		<tr>
 			<td align="left"><div class="required" title="<?php echo _AC('required_field'); ?>">*</div><label for="form_password1"><?php echo _AC('password'); ?></label>:</td>
-			<td align="left"><input id="form_password1" name="form_password1" type="password" size="15" maxlength="15" /></td>
+			<td align="left"><input id="form_password1" name="form_password1" type="password" size="50"  /></td>
 		</tr>
 
 		<tr>
 			<td colspan="2" align="left"><small>&middot; <?php echo _AC('combination'); ?><br />
-				   &middot; <?php echo _AC('15_max_chars'); ?></small></td>
+				   &middot; <?php echo "8 characters minimum"; ?></small></td>
```

Comment:
_AC() is fetching text from the database table language_text.
So, I could change it in code only.

Reply:
The point of using _AC() is to be able to support multiple languages. Hardcoding texts to the page will defeat this purpose. The proper way to modify texts is to change their values saved in the `language_text` table, for example, line 2923 in [the language_text database script](https://github.com/inclusive-design/AChecker/blob/master/install/db/language_text.sql#L2923).

Also, please adjust the `term` if it no longer makes sense, such as `15_max_chars`. Thanks.

Reply:
Oh Yes. I got that now.
Thank You for your guidance.
Is it all done on this issue?
I'll move to the next :)